gradle/profiles/profile-devops.gradle.kts

```diff
@@ -0,0 +1,71 @@
+/* ************************************************* */
+/* default profile                                   */
+/* ************************************************* */
+
+// Truststore configuration
+val truststorePath by extra("/com/forgerock/securebanking/truststore.jks")
+val truststorePassword by extra("changeit")
+/**
+ * OB configuration: Used Create a Software Statement Assertion (SSA) using an API call for Dynamic Registration
+ * https://openbanking.atlassian.net/wiki/spaces/DZ/pages/1322979574/Open+Banking+Directory+Usage+-+eIDAS+release+Production+-+v1.9
+ */
+// OB directory organisation
+val obOrganisationId by extra("0015800001041REAAY")
+val obSoftwareId by extra("ebSqTNqmQXFYz6VtWGXZAa")
+
+// OB token scopes
+val scopesTpp by extra("ASPSPReadAccess TPPReadAccess AuthoritiesReadAccess")
+val scopesAspsp by extra("ASPSPReadAccess TPPReadAll AuthoritiesReadAccess")
+
+/*
+ OB Sandbox directory
+ */
+val obSandboxHostSufix by extra("openbankingtest.org.uk")
+val tokenUrlSandbox by extra("https://matls-sso.$obSandboxHostSufix/as/token.oauth2")
+val testUrlSandbox by extra("https://matls-api.$obSandboxHostSufix/scim/v2/OBAccountPaymentServiceProviders/")
+val audienceSandbox by extra("https://matls-sso.$obSandboxHostSufix/as/token.oauth2")
+val ssaMatlsUrlSandbox by extra("https://matls-dirapi.$obSandboxHostSufix/organisation/tpp/{org_id}/software-statement/{software_id}/software-statement-assertion")
+val ssaMatlsLegacyUrlSandbox by extra("https://matls-ssaapi.$obSandboxHostSufix/api/v1rc2/tpp/{org_id}/ssa/{software_id}")
+/*
+ OB Directory api endpoints
+ */
+val obHostSufix by extra("openbanking.org.uk")
+val tokenUrl by extra("https://matls-sso.$obHostSufix/as/token.oauth2")
+val audience by extra("https://matls-sso.$obHostSufix/as/token.oauth2")
+val testUrl by extra("https://matls-api.$obHostSufix/scim/v2/OBAccountPaymentServiceProviders/")
+val ssaMatlsLegacyUrl by extra("https://matls-ssaapi.$obHostSufix/api/v1rc2/tpp/{org_id}/ssa/{software_id}")
+val ssaMatlsUrl by extra("https://matls-dirapi.$obHostSufix/organisation/tpp/{org_id}/software-statement/{software_id}/software-statement-assertion")
+
+/**
+ * Functional tests configuration
+ */
+// servers
+val environment by extra ("devops")
+val rsServer by extra("https://rs.$environment.forgerock.financial")
+val platformServer by extra("https://iam.$environment.forgerock.financial")
+val amCookieName by extra("80d4a6cdee74c6e")
```

Comment:
I assumed the devops env will use a CDK?
If so then this needs to be iPlanetDirectoryPro

Reply:
That's sorted, thank you